src/app/shared/ui/power-icon/power-icon.html

```diff
@@ -0,0 +1,17 @@
+<div [ngClass]="[themeClass, sizeClass]">
+    <div class="power-icon-info">
+        <div>
+            <p class="amountOfPeople">{{amountOfPeople}}</p>
+            <p class="separetor">/</p>
+            <p class="limitOfPeople" *ngIf="limitOfPeople; else elseBlock">
+                {{limitOfPeople}}
+            </p>
+            <ng-template #elseBlock
```

Comment:
Może zamiast ```#elseBlock``` nazwij to ``` #icon```, albo czymś co by sugerowało czym ta opcjonalna rzecz jest.

Reply:
W pelni zgoda